init-data-repo/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Init Dashboard Data Repo
```

Comment:
Same for this action, it's currently commented out because QA standards is looking into some issues

Reply:
same here as well, will be here and we can determine its usage based on the need as and when it arises